### ISSUE TITLE
ColumnToggler: Erroneous "aria-checked" evaluation

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#5773]
ColumnToggler evaluates the attribute "aria-checked" during handling of the change event. 
https://github.com/primefaces/primefaces/blob/87e23f5ddeb004eea6cd4d185090c1ad8180f06c/src/main/resources/META-INF/resources/primefaces/columntoggler/columntoggler.js#L230

As this attribute is a String it should be fixed by using `$(this).attr('aria-checked') === "true"` instead.
# 1) Environment

PrimeFaces version: 8

# 2) Expected behavior
Correct boolean evaluation on "aria-checked" attribute.
# 3) Actual behavior
The result always evaluates to true
# 4) Steps to reproduce
N/A
# 5) Sample XHTML
N/A
# 6) Sample bean
N/A

**Comment by melloware** [comment]
@MarkusChur You want me to fix this or were you submitting a PR?

**Comment by melloware** [comment]
PR submitted.

**Comment by MarkusChur** [comment]
Thanks @melloware.
